clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy

```diff
@@ -52,10 +55,10 @@ import java.security.Security
   CloudFoundryConfig,
   AzureConfiguration,
   SecurityConfig,
-  com.netflix.spinnaker.kato.Main,
   com.netflix.spinnaker.mort.Main,
   com.netflix.spinnaker.oort.Main
 ])
+@ComponentScan(["com.netflix.spinnaker.kato.config"])
```

Comment:
Gotcha.

I reckon we can remove this once all the kato-XXX providers are migrated as the @Import(XYZConfig) should add an appropriate component scan.